File: src/main/java/com/softwareloop/glo/model/Datafile.java
package com.softwareloop.glo.model;

import lombok.Data;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;

@Data
@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class Datafile {

    @XmlElement
    private Header header;

    @XmlElement(name = "game")
    private List<Game> games;

}

File: src/main/java/com/softwareloop/glo/dat/model/Datafile.kt
package com.softwareloop.glo.dat.model

import jakarta.xml.bind.annotation.XmlAccessType
import jakarta.xml.bind.annotation.XmlAccessorType
import jakarta.xml.bind.annotation.XmlElement
import jakarta.xml.bind.annotation.XmlRootElement

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
class Datafile {

    @XmlElement
    val header: Header? = null

    @XmlElement(name = "game")
    val games: List<Game>? = null

}